Code/Framework/AzToolsFramework/AzToolsFramework/Application/ToolsApplication.cpp

```diff
@@ -281,6 +283,7 @@ namespace AzToolsFramework
                 azrtti_typeid<Components::EditorLevelComponentAPIComponent>(),
                 azrtti_typeid<Components::EditorEntityActionComponent>(),
                 azrtti_typeid<Components::PropertyManagerComponent>(),
+                azrtti_typeid<AzNetworking::NetworkingSystemComponent>(),
```

Comment:
This should be removed.
The AzNetworkingModule should override the `GetRequiredSystemComponents` and add the NetworkingSystemComponent as required.